### DISCUSSION TITLE
how to switch a wallet

### DISCUSSION THREAD

**Discussion by zhouop0** [btcsuite/btcd#2054] [General]
<img width="972" alt="image" src="https://github.com/btcsuite/btcd/assets/11733741/39e65793-888a-4861-8140-04b78600bb3d">
I run a bitcoin core with signet and I have two wallets.
And rpc client is my local server.
i use above core to get listunspent.

It always throw an expection:
<img width="1062" alt="image" src="https://github.com/btcsuite/btcd/assets/11733741/53a0ddeb-b873-413c-b901-7ba7fa5335f1">

./bin/bitcoin-cli -rpcwallet=xx listunspent 

In ternermal, I can use this commond get the details.

how can I config this wallet path in code?

**Comment by Eoous** [comment]
You could unload all other wallets, then load this walelt.

**Reply by zhouop0** [reply]
I know how to do this.
<img width="589" alt="image" src="https://github.com/btcsuite/btcd/assets/11733741/dd0a7139-feef-46ef-8c7c-07c298e9a532">
host + "/wallet/<walletname>"  

it worked

**Reply by Eoous** [reply]
Yep, it could also work well.
